src/Etg.Yams.Client/YamsClient.cs

```diff
@@ -29,6 +29,8 @@ public class YamsClient : IYamsClient
 
         public async Task Connect()
         {
+            AppDomain.CurrentDomain.UnhandledException += (s, e) => Trace.TraceError(FormatMessage($"App terminated due to unhandled exception: {e.ExceptionObject}"));
```

Comment:
Hmm OrleansClient does this and I've been the one complaining about it :) It should be up to the app to decide what to do with unhandled exceptions. Maybe they have a reason not to log it..